src/style.css

```
@@ -0,0 +1,91 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: #fff;
+}
+
+.block {
+  height: 300px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-basis: 300px;
+  flex-grow: 1;
+  text-align: center;
+}
+
+/* #region blocks bgColor */
+
+.block--1 {
+  background-color: #f00;
+}
+
+.block--2 {
+  background-color: #c00;
+}
+
+.block--3 {
+  background-color: #900;
+}
+
+.block--4 {
+  background-color: #600;
+}
+
+.block--5 {
+  background-color: #300;
+}
+
+.block--6 {
+  background-color: #000;
+}
+
+/* #endregion */
+
+@media screen and (max-width: 599px) {
+  .container {
+    flex-direction: column;
+  }
+}
+
+@media screen and (min-width: 599px) and (max-width: 900px) {
```

you don't need this max-width rule
```suggestion
@media screen and (min-width: 599px) {
```